src/gleam/list.gleam

```diff
@@ -248,6 +250,67 @@ pub fn rest(list: List(a)) -> Result(List(a), Nil) {
   }
 }
 
+fn insert_to_group(
+  groups: Map(k, List(v)),
+  new_key: k,
+  new_value: v,
+) -> Map(k, List(v)) {
+  map.update(
```

Comment:
Let's use `get` and `insert` rather than `update`, that way we don't need to import `option` and the change is much smaller.